pkgs/development/python-modules/betterproto/default.nix

```
@@ -0,0 +1,83 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pythonOlder
+, poetry-core
+, grpclib
+, python-dateutil
+, dataclasses
+, black
+, jinja2
+, isort
+, python3
+, pytestCheckHook
+, pytest-asyncio
+, pytest-mock
+, tomlkit
+, grpcio-tools
+}:
+let
+  version = "2.0.0b5";
+  # The package is separated into a runtime library and a compiler that
+  # compiles proto files into python code. For the compiler, the optional
+  # dependencies are required. If you only need the runtime library, they can
+  # be omitted.
+  optional-dependencies.compiler = [
+    black
+    jinja2
+    isort
+  ];
+in
+buildPythonPackage {
+  pname = "betterproto";
+  inherit version;
```

```suggestion
}:

buildPythonPackage rec {
  pname = "betterproto";
  version = "2.0.0b5";

```

optional-dependencies mirror upstream extra dependencies and normally do not require extra documentation.